uaclient/api/u/unattended_upgrades/status/v1.py

```diff
@@ -145,6 +155,21 @@ def status() -> UnattendedUpgradesStatusResult:
 
 
 def _status(cfg: UAConfig) -> UnattendedUpgradesStatusResult:
+    if not apt.is_installed("unattended-upgrades"):
+        return UnattendedUpgradesStatusResult(
```

Comment:
Not sure how viable it is or maybe it is the expected behaviour, will this have the return code != 0 after it is run on the cli?

Reply:
No, it won't @CalvoM and I guess we don't want it to return non-zero, because it is not an error per se